buildSrc/build.gradle.kts

```diff
@@ -0,0 +1,7 @@
+plugins {
+  `kotlin-dsl`
+}
+
+repositories {
+  mavenCentral()
+}
```

Comment:
What part of the build specifically uses this for fetching dependencies?

Reply:
It's used for the `kotlin-dsl` plugin. Probably for this `kotlin-compiler-embeddable`

Reply:
This is the official documentation for it
```
repositories {
    // The org.jetbrains.kotlin.jvm plugin requires a repository
    // where to download the Kotlin compiler dependencies from.
    mavenCentral()
}
```